web/cypress/integration/deployment.spec.js

```diff
@@ -0,0 +1,87 @@
+describe('Deployment', () => {
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('port forward nginx deployment', () => {
+    cy.exec(
+      `kubectl config view --minify --output 'jsonpath={..namespace}'`
```

Comment:
Do you want to put a TODO here or just create an issue about having Octant start with the temporary context as the default as we discussed on slack?